### DISCUSSION TITLE
How can I change the parameters? Since it start autorouting and I can't change them?

### DISCUSSION THREAD

**Discussion by tormyvancool** [freerouting/freerouting#171] [General]
KiCAD 7.0
Freerouting version 1.7.0

So, I click on the button, and the autorouter starts.
The problem is: if ever I want to change the parameters, I can't.
It just runs and does the job.
The parameters windows opens but it's too late. I can't do anything

**Comment by andrasfuchs** [comment]
Do you use the KiCad plugin? If so, that you're right, there is no option to change the default settings for now.

I agree that it would be useful to have a global `.rules` file with the custom settings.

**Reply by tormyvancool** [reply]
> Do you use the KiCad plugin? If so, that you're right, there is no option to change the default settings for now.
> 
> I agree that it would be useful to have a global `.rules` file with the custom settings.

Yes I do confirm: I use the KiCAD Plugin.
It would be nice that when the autorouter is launched, before to autoroute, enables the user to adjust the parameters (if needed) and then by clicking on OK or Go, it starts the process.

**Comment by Blauschirm** [comment]
This is crippling the KiCad plugin massivly and would be amazing if made possible.
